Tella/Scenes/Authentication/ViewModel/LockViewModel.swift

```diff
@@ -74,7 +70,22 @@ class LockViewModel: ObservableObject {
     }
     
     func removeFilesAndConnections () -> Void {
-        appModel.removeAllFiles()
-        appModel.deleteAllServersConnection()
+        let fileManager = FileManager.default
```

Comment:
@ValbuenaG  Could you add a function in Vaultmanager that can delete the content of document folder 
You can use `containerURL` and `filemanager`